misc/extension_api_validation/4.4-stable.expected

```diff
@@ -14,3 +14,17 @@ Validate extension JSON: Error: Field 'classes/DisplayServer/methods/file_dialog
 Validate extension JSON: Error: Field 'classes/DisplayServer/methods/file_dialog_with_options_show/arguments': size changed value in new API, from 9 to 10.
 
 Optional argument added. Compatibility methods registered.
+
+
+GH-104087
+---------
+Validate extension JSON: Error: Field 'classes/OpenXRAPIExtension/methods/register_composition_layer_provider/arguments/0': type changed value in new API, from "OpenXRExtensionWrapperExtension" to "OpenXRExtensionWrapper".
+Validate extension JSON: Error: Field 'classes/OpenXRAPIExtension/methods/unregister_composition_layer_provider/arguments/0': type changed value in new API, from "OpenXRExtensionWrapperExtension" to "OpenXRExtensionWrapper".
+Validate extension JSON: Error: Hash changed for 'classes/OpenXRAPIExtension/methods/register_composition_layer_provider', from 77170538 to 580EBB70. This means that the function has changed and no compatibility function was provided.
+Validate extension JSON: Error: Hash changed for 'classes/OpenXRAPIExtension/methods/unregister_composition_layer_provider', from 77170538 to 580EBB70. This means that the function has changed and no compatibility function was provided.
+Validate extension JSON: Error: Field 'classes/OpenXRAPIExtension/methods/register_projection_views_extension/arguments/0': type changed value in new API, from "OpenXRExtensionWrapperExtension" to "OpenXRExtensionWrapper".
+Validate extension JSON: Error: Field 'classes/OpenXRAPIExtension/methods/unregister_projection_views_extension/arguments/0': type changed value in new API, from "OpenXRExtensionWrapperExtension" to "OpenXRExtensionWrapper".
+Validate extension JSON: Error: Hash changed for 'classes/OpenXRAPIExtension/methods/register_projection_views_extension', from 77170538 to 580EBB70. This means that the function has changed and no compatibility function was provided.
+Validate extension JSON: Error: Hash changed for 'classes/OpenXRAPIExtension/methods/unregister_projection_views_extension', from 77170538 to 580EBB70. This means that the function has changed and no compatibility function was provided.
+
+Switched from `OpenXRExtensionWrapperExtension` to parent `OpenXRExtensionWrapper`. It's fine because child class can be cast safely to parent.
```

Comment:
Changing the parameter types will break binary compatibility for C#. If you don't want to register compatibility methods, at least add overloads in [`Compat.cs`](https://github.com/godotengine/godot/blob/b377562b52fa8a68a99d5e818d8e08ac8d3fd7da/modules/mono/glue/GodotSharp/GodotSharp/Compat.cs).

Reply:
Actually, you're totally right - this part does break compatibility!

I think I copy-pasted that without really looking at, because I was expecting it to complain about a different part, which it looks like it didn't even complain about at all :-)

I'll add the compat functions for this in my next update

Reply:
Fixed in my latest push! Sorry about that